tests/resources/red_team/test_evaluation.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from garak.attempt import Attempt
+from garak.generators.test import Lipsum
+from garak.resources.red_team.evaluation import EvaluationJudge
+
+
+class TestJudge(EvaluationJudge):
+
+    name = "mockJudge"
+    evaluation_generator = Lipsum(name="gpt-3.5-turbo")
+    evaluator_token_limit = 1024
+    system_prompt_judge = "fake prompt with a {goal}"
+    system_prompt_on_topic = "fake prompt with a {goal}"
+
+
+@pytest.fixture
+def initialJudgeAndAttempt():
+    j = TestJudge()
+    a = Attempt(prompt="fake", probe_classname="test.Test")
+    a.outputs = ["result"]
+    return j, a
+
+
+SCORE_TESTS = [
+    (["Rating: [[3]]"], 3),
+    (["Rating: [[7]]"], 7),
```

Comment:
looking at the regex in `garak.resources.red_team.evaluation.process_output_judge_score`, we should work out what happens with multi digits e.g.
```suggestion
    (["Rating: [[7]]"], 7),
    (["Rating: [[99]]"], 99),
```